locales/mr-IN.ftl

```diff
@@ -0,0 +1,38 @@
+Title = Firefox Quantum रिलीझ मोहीम
```

Comment:
please note, all keys need to be lowercase -> "title" instead of "Title". This is the same for all keys in here.